extensions/dependencies/logic_proof.html

```diff
@@ -1,3 +1,4 @@
+<!-- TODO(gvishal): Add extension resource prefix here -->
```

Comment:
This one is somewhat problematic, actually. Do you think it's possible to get rid of it altogether and move it to the HTML file for the LogicProof extension? Would that solve the problem?